### PR TITLE
fixing the order of targetFrameworks to show appropriate files in the VS

### PR DESCRIPTION
**Pull request by Anipik**
VS only shows files according to the first TargetFramework.
So replacing netstandard* with netcoreapp5.0

### CODE REVIEW

**Review by safern — COMMENTED** [review]
Are this all the projects?

Also, is this the right fix? Is there a way we can tell VS which Tfm to use, based on best target framework?

**Review by ViktorHofer — APPROVED** [review]
I'm fine with this to unblock others but it would be great if we could investigate if there are other options to achieve this.

File: src/libraries/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <SkipIncludeNewtonsoftJson>true</SkipIncludeNewtonsoftJson>
     <TestRuntime>true</TestRuntime>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
-    <TargetFrameworks>$(NetFrameworkCurrent);$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)$(NetFrameworkCurrent)</TargetFrameworks>
```
Comment (ViktorHofer) [inline src/libraries/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj]: missing semicolon

**Review by safern — APPROVED** [review]
LGTM as well, but agree we should find another solution to this.

**Comment by Anipik** [comment]
Failures are dotnet/core-eng#8971 merging this one

**Comment by safern** [comment]
@Anipik could you open an issue with the concerns above? this is pretty fragile, if someone for some reason adds the latest netcoreapp or whatever at the end instead, that will break that project. We should be resilient to human changes that don't know this clause.

**Comment by Anipik** [comment]
> @Anipik could you open an issue with the concerns above? this is pretty fragile, if someone for some reason adds the latest netcoreapp or whatever at the end instead, that will break that project. We should be resilient to human changes that don't know this clause.

We have a similar issue https://github.com/dotnet/runtime/issues/31902

**Comment by safern** [comment]
We also have: https://github.com/dotnet/runtime/issues/31935

Which one did this fix? Should we close one or the other?

**Comment by Anipik** [comment]
https://github.com/dotnet/runtime/pull/31948 this pr has some discussion over some of the fixes that could be done here. in case anybody is interested.

**Comment by joperezr** [comment]
One last thing here is that we should also fix the target `UpdateVSConfigurations` to make sure that it will always put the most applicable to netcoreapp5.0-WIndows first. Or else, next time somebody runs that this fix will probably just get reverted.

**Comment by safern** [comment]
> One last thing here is that we should also fix the target UpdateVSConfigurations to make sure that it will always put the most applicable to netcoreapp5.0-WIndows first. Or else, next time somebody runs that this fix will probably just get reverted.

I believe that task/target is gone, but not sure.

**Comment by ViktorHofer** [comment]
Yes we don't have that target and task anymore as there's no need for it as we don't encode target frameworks in the sln anymore.

**Pull request merged**